client/core.lisp

```diff
@@ -289,22 +359,16 @@
                                             summary
                                             description)))))
            (arguments-collector (generate-arguments-collector params-spec))
-           (result (list
+           (result (cons
                     `(defgeneric ,name (client ,@generic-lambda-list)
                        ,@generic-body)
-                    
-                    `(defmethod ,name ((client ,class-name) ,@lambda-list)
-                       (let* ((args ,arguments-collector))
-                         (labels ((retrieve-data (args)
-                                    (let ((raw-response (rpc-call client ,original-name args)))
-                                      ,result-transformation)))
-                           (retrieve-data args)))))))
+                    (generate-defmethod name class-name lambda-list arguments-collector
```

Comment:
Probably it is better to rename `generate-defmethod` to `generate-defmethods` because now it returns a list.